package TryCatchBlock;

public class Arithmatic extends Exception {

}
